src/foam/u2/view/FilteredReferenceView.js

```diff
@@ -0,0 +1,93 @@
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'FilteredReferenceView',
+  extends: 'foam.u2.View',
+
+  requires: [
+    'foam.u2.ReferenceView',
+  ],
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  exports: [
+    'filteredDAO'
```

Comment:
Does this need to be exported?

Reply:
In the current setup, no. But if we want to allow somebody to override the filtered view, we need a way for them to refer to the filtered view so you'd be able to do something like:
```
secondView: {
  class: 'some.custom.ReferenceView',
  daoKey: 'filteredDAO'
}
```
But maybe that's not worth caring about yet? What do you think?